src/main/java/org/cactoos/text/ComparableText.java

```diff
@@ -69,4 +70,24 @@ public String asString() throws Exception {
         return this.text.asString();
     }
 
+    @Override
+    public boolean equals(final Object other) {
```

Comment:
@ilyakharlamov Please, use `Equals` instead.

Reply:
@fabriciofx 
Sorry, I am not familiar with the Equals object, how is it supposed to be used if the object you compare it against is `Object`, not `Scalar`?
I tried searching for some doc and description couldn't find any relevant example of such a case.
is that supposed to be an unsafe cast like ```new Equals(this.field, (Scalar) other)``` or what?

Reply:
@ilyakharlamov If there isn't documentation enough to understand `Equals` usage it can be a bug. Can you open an issue about it? Thanks!

Reply:
@ilyakharlamov You can use something like: `new Equals(() 0-> this.field1, () -> this.field2).value()` or something like that. WDYT?

Reply:
Sorry, I tried to find, but don't understand how to use `new Equals` in the `boolean equals(Object other)` method. There are a few problems:
- `new Equals<>(
                    () -> ...Text
                    () -> ...Text
                )` obviousy does not compile.

Reply:
@ilyakharlamov You can't use a `UncheckedScalar<>(new Equals(...)).value()` (second option). You can use cast, if necessary.

Reply:
@ilyakharlamov Let's wait for [cactoos-matchers](https://github.com/llorllale/cactoos-matchers/issues/90).